frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/_add-capacity-modal.scss

```diff
@@ -5,28 +5,22 @@
 }
 
 .add-capacity-modal--padding {
-  .co-storage-class-dropdown {
-    .dropdown {
-      width: 21.5rem;
-        button {
-          width: 100%;
-        }
-    }
-  }
-  .form-group input {
-    max-width: 17rem;
-    margin-right: 5px;
-  }
-  .toolTip_dropdown {
-    button {
-      position: absolute;
-      left: 6.6rem;
-      top: 8.5rem;
-    }
-  }
   padding-top: 2em;
 }
 
+.add-capacity-modal--padding .co-storage-class-dropdown .dropdown {
```

Comment:
Instead of relying on the nesting, it would be better to update the storage class dropdown to take a `className` attribute and then give it a class name that follow BEM naming. Then these rules become a lot simpler.

Reply:
It's unclear to me why we need special styles here, though. We have dropdowns in other modals and forms. We should be able to reuse existing styles.

Reply:
@spadgett according to the UX design, the storage-dropdown should be shorter than used in other views, that's why applied this style. Have added a BEM naming as suggested.

Reply:
> @spadgett according to the UX design, the storage-dropdown should be shorter than used in other views, that's why applied this style.

Can you attach a screenshot? We should try to be consistent in most cases.

Reply:
Ya sure,
![Screenshot from 2019-11-15 11-42-28](https://user-images.githubusercontent.com/12200504/68921029-4f4e1a80-079d-11ea-8e33-96245f2f489c.png)